addons/website_sale/tests/test_delivery_ui.py

```diff
@@ -41,7 +40,7 @@ def test_01_free_delivery_when_exceed_threshold(self):
             'name': 'The Poste',
             'sequence': 9999, # ensure last to load price async
             'fixed_price': 20.0,
-            'delivery_type': 'base_on_rule',
+            'delivery_type': 'fixed',
```

Comment:
This change is because 'base_on_rule' was moved in stock_delivery in a previous version. I've undone this one. Other change is because the variable is declared but not used.